static/js/app/cloudfoundry.js

```diff
@@ -0,0 +1,57 @@
+(function() {
+    // CloudFoundry Service
+    var cloudfoundry = function($http) {
+
+        // Get current authentication status from server
+        var getAuthStatus = function() {
+            return $http.get('/v2/authstatus')
+                .then(function(response) {
```

Comment:
is this like success and failure? if they do they the same thing, do we need two different anon functions?